apps/src/code-studio/components/ShareDialog.jsx

```diff
@@ -165,6 +201,11 @@ const ShareDialog = React.createClass({
       modalClass += ' no-modal-icon';
     }
 
+    const hasThumbnail = this.props.thumbnailUrl;
```

Comment:
nit: better to coerce this to a boolean by doing `!!this.props.thumbnailUrl`, since the variable you're assigning it to is named like one